SPECS/iptables/ip4save

```diff
@@ -1,13 +1,12 @@
 # init
 *filter
-:INPUT DROP [0:0]
-:FORWARD DROP [0:0]
-:OUTPUT DROP [0:0]
-# Allow local-only connections
+:INPUT ACCEPT [0:0]
+:FORWARD ACCEPT [0:0]
```

Comment:
Accepting FORWARD by default isn't a good idea for machines that are going to be on the internet.